lib/ibm/cloud/sdk/iam.rb

```diff
@@ -1,35 +1,36 @@
+# frozen_string_literal: true
+
 module IBM
   module Cloud
     module SDK
+      # Get a new Bearer using the API Key.
       class IAM < BaseService
         def endpoint
-          "https://iam.cloud.ibm.com".freeze
+          'https://iam.cloud.ibm.com'
         end
 
-        def initialize(api_key)
+        def initialize(api_key) # rubocop:disable Lint/MissingSuper
```

Comment:
Really don't like "linter" directives cluttering code, we should be able to just ignore rubocop when we know it is wrong.  The rules are more like guidelines anyway :laughing:

Reply:
This comment disables the linter for this line only. It represents a knowing violation of the linter rule, and is easier to work with when listing in the IDE.